tools/distrib/yapf_code.sh

```diff
@@ -53,7 +53,7 @@ for dir in $DIRS; do
   tempdir=`mktemp -d`
   cp -RT $dir $tempdir
   $PYTHON -m yapf -i -r -p $exclusion_args $dir
-  if ! diff -rq $dir $tempdir; then
+  if ! diff -r $dir $tempdir; then
```

Comment:
1. Wouldn't we like to learn about all the problems in a single test run rather than learn about problems in one directory, fix them, learn about problems in another directory, fix them, learn about problems in a third directory, fix them, and so on?
2. How certain are we that we want to keep the temporary directory contents?